src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +96,23 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return someDate.isAfter(LocalDate.now()) ? someDate + " is after " + LocalDate.now()
```

Comment:
create a variable in this method not to call LocalDate.now multiple times

Reply:
Done